tests/pytest.ini

```diff
@@ -33,3 +34,4 @@ filterwarnings =
     ignore:PauliWord.hamiltonian:pennylane.PennyLaneDeprecationWarning
 addopts = --benchmark-disable
 xfail_strict=true
+rng_salt = v0.39.0
```

Comment:
I'm using the stable version of PL as the global salt, with the understanding that updating the seed once every release cycle sounds like a reasonable frequency to do so. Although the current stable version is `v0.38.0`, we are about to release `v0.39.0` in two weeks. The next time we update this salt should probably be after the release of `v0.40.0`.